app/src/main/java/org/wikipedia/util/DateUtil.kt

```diff
@@ -82,8 +82,9 @@ object DateUtil {
         return getDateStringWithSkeletonPattern(date, "yyyy")
     }
 
-    fun getYMDDateString(date: Date): String {
-        return getCachedDateFormat("yyyyMMdd", Locale.ROOT, true).format(date)
+    fun getYMDDateString(date: LocalDate): String {
```

Comment:
This necessitated updating this function to use `java.time` (better), which also meant updating its usage in EditHistoryListViewModel.